bc-ipfs/src/FileList.js

```diff
@@ -32,12 +32,15 @@ class FileList extends Component {
       targetSearchTemplate = 'blockmed-trans-aggs';
     }
 
-    window.fetch('http://es.comehere.app/blockmed-trans-*/_search/template', {
+    window.fetch('http://es.blcksync.info/blockmed-trans-*/_search/template', {
       method: 'POST',
       headers: {
         'Authorization': 'Basic YWRtaW46YWRtaW4=',
-        'Content-Type': 'application/json'
+        'Content-Type': 'application/json',
+        'Access-Control-Allow-Origin': '*'
```

Comment:
```
'Access-Control-Allow-Origin': '*'
```
`Access-Control-Allow-Origin` should be in server-side response, not in the client request, so client browser (Chrome) can detect this value for security control.